cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -102,6 +102,7 @@
       cronjob at:'*/30 * * * *', do:dashboard_dir('bin', 'sync_jotforms')
       cronjob at:'0 6 * * *', do:deploy_dir('bin', 'cron', 'process_foorm_data')
       cronjob at:'25 7 * * *', do:deploy_dir('bin', 'cron', 'update_hoc_map')
+      cronjob at:'49 5 * * *', do:deploy_dir('bin', 'cron', 'update_census_mapbox')
```

Comment:
@davidsbailey Was just discussing this with Vijaya and I'm wondering if this cronjob frequency should be more infrequent than daily since this is theoretically something we only do once a year (plus perhaps some manual adjustments to values if the need arose)? Curious on your thoughts!

Reply:
Sounds good! I've moved the script to the bin directory and updated the dev doc to remove "cronjob" wording and say that the script must be run on production after the changes are merged. What do you mean by "next step of manually updating the `censustiles` tileset which is visible in prod"? Wouldn't we want to keep the production tileset as `censustiles` and just run the script on production once the changes are merged?

Reply:
that sounds like a good way to do it! 

what I meant was, after validating locally, run the script locally again to update the `censustiles` tileset.

Reply:
Oh gotcha yes that makes sense!